### PR TITLE
added navigation component

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Review navigation component

PREVIEW: https://vercel-ui-preview.netlify.app/component-lib/navigation.html

### CODE REVIEW

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
You can give some left and right padding to nav, as user profile looks too close to corner.

**Comment by nikhiljugale007** [comment]
> You can give some left and right padding to nav, as user profile looks too close to corner.

added 16 px padding to left and right.

**Comment by anasm11** [comment]
Good design. Looks good to merge.

File: component-lib/navigation.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                                    sandbox="allow-scripts allow-same-origin">
+                                </iframe>
+							</code>
```
Comment (Ankur9669) [inline component-lib/navigation.html:245]: You can give left right margin to your iframe, because on mobile screens it is sticked to both the sides

**Review by Ankur9669 — APPROVED** [review]
Awesome!!!. The navbar is looking great you can go ahead and merge it

File: component-lib/navigation.html (hunk 1 above)
Comment (nikhiljugale007) [reply component-lib/navigation.html:245]: added that in other PR (slider component PR ) it will get updated once I merge that. Thanks.

**Pull request merged**